lib/cc/engine/markdownlint.rb

```diff
@@ -5,6 +5,8 @@
 module CC
   module Engine
     class Markdownlint
+      EXTENSIONS = %w[.markdown .md].freeze
```

Comment:
Thanks! I punted here and just went with `.md` since that was `mdl` did, but this is better.